it/v1/endpoints/AuthISpec.scala

```diff
@@ -75,7 +75,7 @@ class AuthISpec extends IntegrationBaseSpec {
 
         override def setupStubs(): StubMapping = {
           AuditStub.audit()
-          MtdIdLookupStub.internalServerError(nino)
+          MtdIdLookupStub.error(nino, Status.INTERNAL_SERVER_ERROR)
```

Comment:
Can you import INTERNAL_SERVER_ERROR so that "Status." can be removed. Same for all of these occurrences.